Jenkinsfile

```diff
@@ -195,15 +195,20 @@ pipeline {
         stage("provision test") {
             when {
                 expression {
-                    return pullRequestContainsLabels("test-provision,test-provision-aws,test-provision-gce,test-provision-docker,test-provision-k8s-local-kind-aws,test-provision-k8s-eks,test-provision-azure")
+                    def labels = [
```

Comment:
Not critical, but readability may be improved here by following structuring:

```
def labels = [
	"test-provision",
	"test-provision-docker",
	"test-provision-aws", "test-provision-aws-reuse",
	"test-provision-gce", "test-provision-gce-reuse",
	"test-provision-k8s-local-kind-aws", "test-provision-k8s-local-kind-aws-reuse",
	"test-provision-k8s-eks", "test-provision-k8s-eks-reuse",
	"test-provision-azure", "test-provision-azure-reuse"
].join(",")
```

Reply:
Changed the list for readability
The sanity check that pipeline is working - [test-provision-aws-reuse](https://jenkins.scylladb.com/job/sct-github-PRs-scan/job/scylla-cluster-tests/job/PR-8405/25/)